src/api/AuthServer/AuthServer/Mapping/MappingProfile.cs

```diff
@@ -0,0 +1,17 @@
+﻿using AuthServer.DTO;
+using AutoMapper;
+
+namespace AuthServer.Mapping
+{
+    public class MappingProfile : Profile
+    {
+        public MappingProfile()
+        {
+            CreateMap<UserRegisterDTO, ApplicationUser>();
+            CreateMap<ApplicationUser, UserRegisterDTO>();
+
+            CreateMap<ApplicationUser, UserDTO>();
+            CreateMap<UserDTO, ApplicationUser>();
```

Comment:
Why do we need 2-way mapping here? Why would we map `ApplicationUser` to `UserRegisterDTO`?